afs/templates/views/components/reports/place.htm

```diff
@@ -120,6 +124,16 @@ <h1 class="afs-summary-report-title"><span data-bind="text: displayname" class="
                     </table>
                 </div>
                 <!-- /ko -->
+            </div>
+        </div>
+        <!-- Geospatial Definition -->
+        <div class="afs-report-section">
+            <h2 class="afs-report-section-title">{% trans "Geospatial Definitions" %}</h2>
+            <div class="afs-report-subsection" >
+                <!-- ko if: geojson == '--' -->
+                <h3 class="afs-report-nodata-message">{% trans "There's no geospatial definition for this resource" %}</h3>
```

Comment:
For consistency, let's not use contractions.  So "There is no geospatial definition for this resource."

Reply:
Fixed.